examples/chip-tool/commands/common/Commands.cpp

```diff
@@ -138,7 +139,8 @@ CHIP_ERROR Commands::RunCommand(int argc, char ** argv, bool interactive)
         }
     }
 
-    if (!command->InitArguments(argc - 3, &argv[3]))
+    int argumentsPosition = isGlobalCommand ? 4 : 3;
+    if (!command->InitArguments(argc - argumentsPosition, &argv[argumentsPosition]))
```

Comment:
So for the cases when we have a "global" command, the above will consumer the event/attr name and the first arg passed to InitArguments will be the first arg after that, right?

Then why does InitArguments need the AttributeOrEventName checks?  Those need to be clearly documented.

Reply:
> So for the cases when we have a "global" command, the above will consumer the event/attr name and the first arg passed to InitArguments will be the first arg after that, right?
> 

This is correct.

> Then why does InitArguments need the AttributeOrEventName checks? Those need to be clearly documented.

You are correct again. I have updated the code such that it is not needed.